.drone/go_wrapper.sh

```diff
@@ -0,0 +1,161 @@
+#!/usr/bin/env bash
```

Comment:
Do we need to copy this from seego into this repository? I'm worried about keeping them in sync.

Reply:
Nope, your correct, I can reference the built in one.